pkg/http/api.go

```diff
@@ -780,27 +781,35 @@ func (a *api) onDirectMessage(reqCtx *fasthttp.RequestCtx) {
 	respond(reqCtx, statusCode, body)
 }
 
-func (a *api) onCreateActorReminder(reqCtx *fasthttp.RequestCtx) {
+func (a *api) getActorWithRequestValidation(reqCtx *fasthttp.RequestCtx, req interface{}) (string, string, error) {
+	var err error
 	if a.actor == nil {
 		msg := NewErrorResponse("ERR_ACTOR_RUNTIME_NOT_FOUND", messages.ErrActorRuntimeNotFound)
 		respondWithError(reqCtx, fasthttp.StatusInternalServerError, msg)
-		return
+		return "", "", errors.New(msg.Message)
 	}
```

Comment:
i think it would be better refactor each parts separately in this function, instead of provide a getActorWithRequestValidation.

for example: 
* generic check (check actor);
* base common attributes from ctx: should we refactor the request struct themselves;
* and other general error check.

Just an idea, for your reference.